ci/infra/azure/security-groups.tf

```diff
@@ -0,0 +1,144 @@
+locals {
+  security_rules_commons = [
+    {
+      "name" : "ssh",
+      "direction" : "Inbound",
+      "access" : "Allow",
+      "protocol" : "Tcp",
+      "source_port_range" : "*",
+      "destination_port_range" : "22",
+      "source_address_prefix" : "*",
+      "destination_address_prefix" : "*"
```

Comment:
I see you changed the format of all the dictionaries. I wonder if `terraform fmt` is not going to complain about that

Reply:
I have ran `terraform fmt`  It didn't complain.